lib/vcr/library_hooks/typhoeus.rb

```diff
@@ -117,4 +135,3 @@ def self.vcr_response_from(response)
     WebMock::HttpLibAdapters::TyphoeusAdapter.disable!
   end
 end
-
```

Comment:
Would you consider using Rubocop to codify & automatically perform such code style checks?